### PR TITLE
Add FPS Demo

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds an FPS Demo page to the sample apps folder.

REVIEWERS - Please make sure the content is correct and complete. I wrote it based on Slack conversations, so I might have missed something.

@lohaniabhishek - We'll need to change the image and probably the text when you get the new version up.

### CODE REVIEW

**Comment by diana-macrometa** [comment]
@lohaniabhishek and @joubert-macrometa, this is the FPS demo PR.

File: docs/apps/fingerprinting-demo.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 40
+title: Fingerprinting Demo
+---
+
+This demo app implements the PhotonIQ Fingerprint Service (FPS) to track site users. It identifies unauthenticated site visitors, even when they switch tabs or clear browsing data.
```
Comment (lohaniabhishek) [inline docs/apps/fingerprinting-demo.md]: this has nothing to do with 'unauthenticated site visitors'

`It identifies anonymous site visits, even if the use incognito or clear browsing data`

**Review by lohaniabhishek — APPROVED** [review]

File: docs/apps/fingerprinting-demo.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/apps/fingerprinting-demo.md]: Based on what you said, I also updated the coupon fraud prevention. Please take a look.

**Comment by joubert-macrometa** [comment]
@lohaniabhishek What is the name of our service?
PhotonIQ FPS
PhotonIQ DS

The reason I ask is in the text I see:
_PhotonIQ Fingerprint Service (FPS)_

But in our UI I see this:
image
I think in this image Photoniq is not capitalized correctly, or am I wrong?
But is the DS part right?

**Comment by diana-macrometa** [comment]
@reviewers, I have updated the link.

@nicolewilliams88 or @macrochrissie, do you want to weigh in on the naming of this? They're calling it PhotonIQ Fingerprint in this demo, not sure that aligns with our branding.

**Comment by macrochrissie** [comment]
> @reviewers, I have updated the link.
> 
> @nicolewilliams88 or @macrochrissie, do you want to weigh in on the naming of this? They're calling it PhotonIQ Fingerprint in this demo, not sure that aligns with our branding.

It's called PhotonIQ Digital Fingerprinting - we reference these as services but we don't have the word service in any of the names.

**Comment by diana-macrometa** [comment]
@macrochrissie, document text is updated. @lohaniabhishek and @rahulprabhune-macrometa, can we update the demo or have Matt do it?

File: docs/apps/fingerprinting-demo.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 40
+title: Fingerprinting Demo
+---
+
+This demo app implements the PPhotonIQ Digital Fingerprinting to track site users. It identifies anonymous site visits, even if the use incognito or clear browsing data.
```
Comment (joubert-macrometa) [inline docs/apps/fingerprinting-demo.md]: Double P...

Comment (joubert-macrometa) [reply docs/apps/fingerprinting-demo.md]: "the PhotonIQ Digital Fingerprinting"
That seems strange wording.

Comment (diana-macrometa) [reply docs/apps/fingerprinting-demo.md]: Good catches. Thanks!

**Pull request merged**